### ISSUE TITLE
Add ability to create pokemon in an egg

### ISSUE THREAD

**Issue by magroader** [Jerakin/Pokedex5E#634]
PHB with Gen VII adds breeding and eggs. Eggs take up Pokeslots, so they should probably be in the app in some way.

A few notes:
* Eggs will likely have the same graphics, so if you have 2+ eggs it will be easy to get mixed up. We may want to consider improving the way a user can SORT their party to make sure they don't lose track of which egg is which. Alternatively, just changing the name of the egg or adding a note to them would work.
* We don't need to implement all the rules, just creating a new pokemon and checking a box for "in egg" may be sufficient.
* Of course there's gotta be a "Hatch" button in the edit screen. No question there.

**Comment by Jerakin** [comment]
Yeah, it would be cool if a DM could "wrap" a Pokemon in an Egg and then send it to a Player. Maybe we can make up some primitive "scambler" of the json data so that the player can't read it and figure out directly what it is!

**Comment by magroader** [comment]
Ooh I love that scrambling idea!
